recipes/cairo/all/conanfile.py

```diff
@@ -99,47 +119,73 @@ def build(self):
+                self.deps_cpp_info["zlib"].libs[0] + ".lib",
+            )
+            tools.replace_in_file(
+                win32_common,
+                "$(LIBPNG_PATH)/lib/libpng16.lib",
+                self.deps_cpp_info["libpng"].libs[0] + ".lib",
+            )
+            tools.replace_in_file(
+                win32_common,
+                "$(FREETYPE_PATH)/lib/freetype.lib",
+                self.deps_cpp_info["freetype"].libs[0] + ".lib",
+            )
             with tools.vcvars(self.settings):
                 env_msvc = VisualStudioBuildEnvironment(self)
-                env_msvc.flags.append('/FS')  # C1041 if multiple CL.EXE write to the same .PDB file, please use /FS
+                env_msvc.flags.append(
+                    "/FS"
+                )  # C1041 if multiple CL.EXE write to the same .PDB file, please use /FS
                 with tools.environment_append(env_msvc.vars):
                     env_build = AutoToolsBuildEnvironment(self)
-                    args=[
-                        "-f", "Makefile.win32",
+                    args = [
+                        "-f",
+                        "Makefile.win32",
                         "CFG={}".format(str(self.settings.build_type).lower()),
                         "CAIRO_HAS_FC_FONT=0",
                         "ZLIB_PATH={}".format(self.deps_cpp_info["zlib"].rootpath),
                         "LIBPNG_PATH={}".format(self.deps_cpp_info["libpng"].rootpath),
-                        "PIXMAN_PATH={}".format(self.deps_cpp_info['pixman'].rootpath),
-                        "FREETYPE_PATH={}".format(self.deps_cpp_info['freetype'].rootpath),
-                        "GOBJECT_PATH={}".format(self.deps_cpp_info['glib'].rootpath)
+                        "PIXMAN_PATH={}".format(self.deps_cpp_info["pixman"].rootpath),
+                        "FREETYPE_PATH={}".format(
+                            self.deps_cpp_info["freetype"].rootpath
+                        ),
+                        "GOBJECT_PATH={}".format(self.deps_cpp_info["glib"].rootpath),
                     ]
 
                     env_build.make(args=args)
-                    env_build.make(args=['-C', os.path.join('util', 'cairo-gobject')] + args)
+                    env_build.make(
+                        args=["-C", os.path.join("util", "cairo-gobject")] + args
+                    )
 
     def _get_env_build(self):
         if self._env_build:
             return self._env_build
 
-        self._env_build = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        self._env_build = AutoToolsBuildEnvironment(
+            self, win_bash=tools.os_info.is_windows
+        )
         yes_no = lambda v: "yes" if v else "no"
         configure_args = [
-            "--datarootdir={}".format(tools.unix_path(os.path.join(self.package_folder, "res"))),
+            "--datarootdir={}".format(
+                tools.unix_path(os.path.join(self.package_folder, "res"))
```

Comment:
What files does it install in the `res` folder?
Are they needed at build time?
Can some folders, such as `res/man` be removed?

Reply:
I'm not sure, this was preexisting code